opentelemetry-sdk/src/metrics/view.rs

```diff
@@ -110,7 +110,7 @@ pub fn new_view(criteria: Instrument, mask: Stream) -> MetricResult<Box<dyn View
     let contains_wildcard = criteria.name.contains(['*', '?']);
 
     let match_fn: Box<dyn Fn(&Instrument) -> bool + Send + Sync> = if contains_wildcard {
-        if !mask.name.is_empty() {
+        if mask.name.is_some() {
```

Comment:
We still might have to check for empty string. A user could still provide an empty string as `Some(String::new())`.